src/pycookiecheat/chrome.py

```diff
@@ -88,6 +88,10 @@ def chrome_decrypt(
     decryptor = cipher.decryptor()
     decrypted = decryptor.update(encrypted_value) + decryptor.finalize()
 
+    if cookie_database_version >= 24:
+        # Cookies in database version 24 and later include a SHA256 hash of the domain to the start of the encrypted value.
```

Comment:
Would you please add a link to source? The one from [your comment](https://github.com/n8henrie/pycookiecheat/issues/72#issuecomment-2431668435) looks static and should be fine.

Reply:
Also see above re: PEP8.